ipatests/test_integration/test_installation.py

```diff
@@ -1178,6 +1258,23 @@ def test_install_master(self):
 
         assert "Administrator e-mail address: me.example.org" in result
 
+    @master_unistalled
+    def test_installer_wizard_should_prompt_for_DNS(self):
+        """
+        Test for #2575 [RFE] Installer wizard should prompt for DNS
+
+        https://pagure.io/freeipa/issue/2575
+        """
+        tasks.install_master(self.master,
```

Comment:
It is not clear if this command really checks anything about installer questions. Also this step can brake and it will be difficult to find out why when the installation scenario (i.e. wizard questions) changes.
I suggest to rewrite this test similar to https://github.com/freeipa/freeipa/blob/886c0dcb07e84e2d92545d84bed4bc55c5c520f7/ipatests/test_integration/test_adtrust_install.py#L108